src/components/Cards/Card.css

```diff
@@ -1,5 +1,6 @@
 :root{
   --badge-bg:#dedfe0;
```

Comment:
maybe move this to common css file, it will be helpful in another component as well

Reply:
kept here because cards are used almost everywhere